vast/integration/vast_integration_suite.yaml

```diff
@@ -819,3 +819,40 @@ tests:
         input: data/json/empty-record.json
       - command: exec 'from - read json | write xsv to stdout'
         input: data/json/empty-record.json
+
+  Zeek TSV Pipeline Format:
+    tags: [pipelines, zeek]
+    steps:
+      - command: exec 'from stdin read zeek-tsv | write json to stdout'
+        input: data/zeek/merge.log
+      - command: exec 'from stdin read zeek-tsv | write json to stdout'
+        input: data/zeek/merge_with_whitespace_separation.log
+      - command: exec 'from stdin read zeek-tsv | write zeek-tsv to stdout'
+        input: data/zeek/dns.log.gz
+      - command: exec 'from stdin read zeek-tsv | write csv to stdout'
+        input: data/zeek/dns.log.gz
+      - command: exec 'from stdin read zeek-tsv | write zeek-tsv ; empty NULLVAL to stdout'
```

Comment:
What's the semicolon separation doing?

Reply:
The `;` is the optional  `set_separator` argument for the output.

Reply:
I think this is quite difficult to read wrt. precedence. No way I would have thought that `;` is the argument to `zeek-tsv`! Using positional arguments here is an issue also because of `from ... to ...`, as it completely shadows the `to` part. From a UX side, I think want `--key=value` flags.

Reply:
For `xsv` the positional arguments are mandatory and cannot be omitted. For `zeek-tsv` they are options, so they should be passed in as `--option=value`.

Reply:
I made use of `option_set_parser` in https://github.com/tenzir/vast/pull/3114/commits/8a98b3bd37acd143aa26cc39d2629950ed74c628